client/src/components/EditProfileDialog.js

```diff
@@ -63,6 +63,11 @@ const styles = theme => ({
             textTransform: "initial"
         }
     },
+    loadingButton: {
+        color: theme.palette.grey[500],
+        disabled: true,
+        padding: "4px 4px 4px 22px"
```

Comment:
Do you remember we talked about buttons getting their padding from theme.js? :) You can remove this custom padding and button will look the same